sdk/python/kfp/dsl/_metadata.py

```diff
@@ -221,6 +227,14 @@ def _extract_pipeline_metadata(func):
     arg_default = arg_defaults[arg] if arg in arg_defaults else None
     if arg in annotations:
       arg_type = _annotation_to_typemeta(annotations[arg])
+    if 'openapi_schema_validator' in arg_type.properties and arg_default is not None:
+      from jsonschema import validate
+      import json
+      schema_object = arg_type.properties['openapi_schema_validator']
```

Comment:
What do you think about renaming `openapi_schema_validator` to just `json_schema`? We haven't really exposed this to the users, so we can still do that.
`json_schema` is more correct term since `{"type": "string"}` is a valid JSON Schema, but not a valid OpenAPI schema.

Reply:
Are you sure that openapi schema does not support {"type":"string"}(https://swagger.io/docs/specification/data-models/data-types/). 

BTW, there are lots of openapi schema validator. For example https://github.com/kogosoftwarellc/open-api/tree/master/packages/openapi-schema-validator is in fact using the json schema validator under the hood.

I'm OK with json_schema_validator. I can send send another PR to change all openapi_schema_validator  to json_schema_validator.

Reply:
Talked offline:
We agreed to rename `openapi_schema_validator` to `json_schema` in another PR.

Reply:
Added an issue: https://github.com/kubeflow/pipelines/issues/1416